app/controllers/bike_share_app.rb

```diff
@@ -1,3 +1,158 @@
+require_relative '../models/station'
```

Comment:
you shouldn't need to require your models. this is being done in `config/environment.rb`